File: modules/recolor.py
#color module made for the people using python :)
def colorText(message,rgbarray):
    return f'\x1b[38;2;{";".join(map(str, rgbarray))}m{message}\x1b[0m' #return text for regular printing
def colorLog(message,rgbarray):
    print(f'\x1b[38;2;{";".join(map(str, rgbarray))}m{message}\x1b[0m') #print text for you
# import recolor
# colorLog("hello", [0,255,0]) #green
# colorText("Hello World!", [255,0,0]) #red
colorLog("hello", [255,0,0])

File: modules/recolor.js
module.exports.colorText = (message,rgbarray)=>{//recolor v2
    return(`\x1b[38;2;${rgbarray.join(";")}m${message}\x1b[0m`)//will not work if truecolor isnt supported
};
module.exports.colorLog = (message,rgbarray)=>{
    console.log(`\x1b[38;2;${rgbarray.join(";")}m${message}\x1b[0m`)
    return true
}